LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 frissyn
+Copyright (c) 2020 IreTheKID
```

Comment:
Is this purposeful? Is the name "IreTheKID" or "Frissyn" preferred? @frissyn 

Also, shouldn't this be something along the lines of 2020-2021?

Reply:
Oh what the heck, my repl didn't pull properly.